ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec-prod.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
same comment here. please talk to Azam and Sandeep about this

Reply:
Having a discussion with both and the change with be in base file as it is not impact the functionality. This file will be removed